corehq/tests/nose.py

```diff
@@ -220,7 +220,8 @@ def _databases_ok(self):
             assert db["NAME"].startswith(TEST_DATABASE_PREFIX), db["NAME"]
             try:
                 connection.ensure_connection()
-            except OperationalError:
+            except OperationalError as e:
+                sys.__stderr__.write(str(e))
```

Comment:
The convention I've seen is

```
print(str(e), file=sys.stderr)
```
(which I guess differs in that it has a newline terminator by default). Is this different from that (other than the newline)?

Reply:
to be exactly the equivalent it would be
```py
print(str(e), file=sys.__stderr__, end="")
```
I think the newline is probably a good thing, so don't use `end=""` if switching to this.

Reply:
also need to add `from __future__ import unicode_literals` at the top

Reply:
Updated: 5eb3064